README.md

```diff
@@ -56,6 +56,10 @@ In Patch, orders represent a purchase of carbon offsets or negative emissions by
 const mass = 1000000 // Pass in the mass in grams (i.e. 1 metric tonne)
 patch.orders.createOrder({ mass_g: mass })
 
+// Create an order with maximum total price
+const totalPriceCentsUSD = 500 // Pass in the total price in cents (i.e. 5 dollars)
```

Comment:
Should we add to the README that you should pick either `mass_g` or `total_price_cents_usd` when creating an order?

Reply:
added a bit more context, see if that looks better!